styles/index.css

```diff
@@ -0,0 +1,533 @@
+@import url('https://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+:root {
+  --color1: #272a31;
+  --color2: #ec5242;
+  --color3: #d3d3d3;
+  --color4: #ee7366;
+  --color5: rgba(73, 77, 90, 0.5);
+  --font2: 'Lato', sans-serif;
+  --font1: 'COCOGOOSE', sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  color: var(--color1);
+  text-decoration: none;
+  box-sizing: border-box;
+  font-family: var(--font2);
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: var(--color3);
+}
+
+#navbar {
+  display: flex;
+  position: fixed;
+  flex-direction: column;
+  transition: all 500ms ease-in-out;
+}
+
+#navbar.active {
+  z-index: 10;
+  width: 100%;
+  height: 100vh;
+  background-color: var(--color3);
+}
+
+#topline {
+  display: none;
+}
+
+.brand-title {
+  display: none;
+}
+
+.toggle {
+  width: 70px;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 1rem 1rem 1rem;
+}
+
+.toggle .bar {
+  margin: 3px;
+  height: 5px;
+  width: 30px;
+  background-color: var(--color2);
+  transition: all 500ms ease-in-out;
+}
+
+.bar1.active {
+  transform: translateY(10.9px) rotate(45deg);
+}
+
+.bar2.active {
+  opacity: 0;
+}
+
+.bar3.active {
+  transform: translateY(-10.9px) rotate(-45deg);
+}
+
+.navbar-links {
+  display: none;
+}
+
+.navbar-links.active {
+  display: block;
+}
+
+.navbar-links ul {
+  display: flex;
+  flex-direction: column;
+}
+
+.navbar-links ul li {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.navbar-links ul li:hover {
+  background-color: var(--color4);
+}
+
+.navbar-links ul li a {
+  width: 100%;
+  font-size: 2rem;
+  text-align: center;
+  color: var(--color1);
+  padding: 1rem 0 1rem 0;
+}
+
+#campaign {
+  display: none;
+}
+
+#home {
+  padding-top: 4rem;
+  background-image: url("../assets/images/background1.png");
+}
+
+#title {
+  padding-left: 1rem;
+  color: var(--color2);
+}
```

Comment:
- [ ] `required`
- [ ] Amazing job so far!! 👏 
- [ ] However, the pages should look almost identical to the original design, 
- [ ] The pages use the exact same `colors`, for `text`, `backgrounds`, `borders`, etc, from the original design
- [ ] Please make the navbar spacing as the original template and reduce the main header
![image](https://user-images.githubusercontent.com/66230306/182006875-248f39ee-b7b4-4276-a63f-26adf2e98175.png) just like the original design

- [ ] see screenshots below for comparisons

`your design`
![headerrr](https://user-images.githubusercontent.com/66230306/182006798-9724b5be-9a99-4f71-8a12-68e70eef41b3.png)

`figma template`
![header shin](https://user-images.githubusercontent.com/66230306/182006827-db79ba63-b77f-494d-8bcc-9b2e576f434e.png)